provisioning/etcd/generate.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+
+cd "$(dirname "$0")"
+helm repo add --force-update bitnami https://charts.bitnami.com/bitnami
+helm template templates-api-etcd bitnami/etcd -f ./values.yaml > etcd.yaml
```

Comment:
ako security best practice by som to radsej oddelil do vlastneho namespace, teda ineho nez je default. Ked uz to ma nejake hesla atd. To by malo ist cez parameter helm template commandu `--create-namespace templates-api-etcd`. Akurat neviem ako to potom vygeneruje network policy, predpokladam ze pody templates-api su v default ns tiez.

Reply:
zas je otazka ked to len kesuje nejake data ci tam je mozne vnimat nejake bezpecnostne riziko.

Reply:
ok, a Templates API dame tiez, do rovnakeho namespace?

Reply:
(kedze do API potrebujem pripojit secret s heslom do etcd)

Reply:
Kacurez: zmenil som vsade namespace z `default` na `templates-api`: https://github.com/keboola/keboola-as-code/pull/770/commits/8095405f19d657384589b555767edbb8adf4a61e

Reply:
to bude lepsie zmazat v deploy PR do mastru cez CI/CD, tj do deploy skriptov sa prida `kubectl delete service -n default templates-api --ignore-not-found` atd.. a v dalsom PR sa ten delete kod odmaze a nemusite to robit rucne.
Este som si vsimol ze nacitanie IP adresy vytvorenej service sa berie z templates-api v default ns. Tam to bude tiez treba zmenit https://github.com/keboola/keboola-as-code/blob/1f462ff616e0cd529bd68a385b49ba5a7d1fb8e1/provisioning/azure/scripts/deploy.sh#L24

Reply:
este sa na to pozriem viac zajtra.